exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/EventDataToAnnotation.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.zipkin;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.trace.data.EventData;
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ * Converts an EventData instance to a String representation of that data, with attributes converted
+ * to JSON.
+ *
+ * <p>See <a
+ * href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md#events">the
+ * zipkin exporter spec</a> for details.
+ */
+class EventDataToAnnotation implements Function<EventData, String> {
+
+  @Override
+  public String apply(EventData eventData) {
```

Comment:
That's fine, making it all static here shouldn't be a big deal. The explicit functional interface wasn't exactly necessary, but it can be helpful to have swappable instances for mocking. In this case, I guess it's fine to just test through the real implementation.

As far as separating -- the OTZSTransformer is already a couple hundred lines long, and I didn't want to add to that. My preference is to separate out clear responsibilities into their own units (SRP).